src/main.ts

```diff
@@ -12,22 +12,22 @@ function getActionInput(): Input {
     repo: getInput('repo') ? getInput('repo') : context.repo.repo,
     packageName: getInput('package-name', { required: true }),
     token: getInput('token', { required: true }),
-    dryRun: getInput('dry-run') ? getInput('dry-run') == 'true' : false,
+    dryRun: getInput('dry-run') ? getInput('dry-run') === 'true' : false
   });
 }
 
 function run(): Observable<boolean> {
   try {
     return deleteVersions(getActionInput()).pipe(
```

Comment:
This is the entrypoint to the action. `getInput` functions above pull in the input values defined by the individual project's workflow yaml ([see parameter doc here](https://github.com/callrail/package-cleanup/blob/main/action.yml#L4-L39)). 

`deleteVersions()` is in delete.ts above, which fetches versions from the Github GraphQL API, checks if they match the input `branch-name` and if so sends a mutation request back to the API to delete the version.